### DISCUSSION TITLE
XSLT Mapper or XSLT to ADM Converter

### DISCUSSION THREAD

**Discussion by ddiegomercado** [atlasmap/atlasmap#4618] [Q&A]
I would like to know if there is a way to edit an existing XSLT using the atlas maps graphical mapping editor? Otherwise, if there is a valid converter from XSLT to ADM,. It would be of great help.

Another option would be to know how to convert XSLT to ADM and make the converter myself.

Thanks a lot in advance.

**Comment by lhein** [comment]
Maybe not a direct answer to your problem but could be worth a consideration. The 2.3 release of Kaoto  (https://kaoto.io) will ship with an integrated datamapper ui which is based on XSLT. That might be an option for you as well.

**Reply by ddiegomercado** [reply]
Thanks a lot, I think it could be a good helper for this use case
